### PR TITLE
[16.0][MIG] l10n_nl_postcode

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

File: l10n_nl_postcode/__manifest__.py [hunk 1]
```diff
@@ -3,12 +3,11 @@
...
     "name": "Dutch postcode validation for Partners",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.0",
     "category": "Localization",
     "author": "Onestein,Odoo Community Association (OCA)",
     "maintainers": ["astirpe"],
     "website": "https://github.com/OCA/l10n-netherlands",
     "license": "AGPL-3",
     "depends": ["base"],
-    "external_dependencies": {"python": ["python-stdnum"]},
```
Comment (bosd) [inline l10n_nl_postcode/__manifest__.py]: Why is this dependency removed?

Comment (astirpe) [reply l10n_nl_postcode/__manifest__.py]: It's already in standard: https://github.com/odoo/odoo/blob/16.0/requirements.txt#L36

Comment (bosd) [reply l10n_nl_postcode/__manifest__.py]: I know, it is a core requirement. Yet, to my understanding it is OCA best practice to still include it in the modules manifest?
Either way, non blocking.

**Review by bosd — APPROVED** [review]
👍 LGTM

**Review by bvanjeelharia — APPROVED** [review]
Looks Good To Merge!!!

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**